tests/init.pp

```diff
@@ -1 +1 @@
-include zypprepo
+include ::zypprepo
```

Comment:
Don't rename tests to examples.  Instead just delete the directory.  The code is a noop.